### ISSUE TITLE
Leftover chroot keys

### ISSUE THREAD

**Issue by StuartIanNaylor** [ayufan-rock64/linux-build#389]
```
W: http://ppa.launchpad.net/ayufan/rock64-0.9-ppa/ubuntu/dists/disco/InRelease: The key(s) in the keyring /etc/apt/trusted.gpg.d/ayufan.key.chroot.gpg are ignored as the file has an unsupported filetype.
W: http://ports.ubuntu.com/ubuntu-ports/dists/disco/InRelease: The key(s) in the keyring /etc/apt/trusted.gpg.d/ayufan.key.chroot.gpg are ignored as the file has an unsupported filetype.
W: http://ports.ubuntu.com/ubuntu-ports/dists/disco-security/InRelease: The key(s) in the keyring /etc/apt/trusted.gpg.d/ayufan.key.chroot.gpg are ignored as the file has an unsupported filetype.
W: http://ports.ubuntu.com/ubuntu-ports/dists/disco-updates/InRelease: The key(s) in the keyring /etc/apt/trusted.gpg.d/ayufan.key.chroot.gpg are ignored as the file has an unsupported filetype.
W: http://deb.ayufan.eu/orgs/ayufan-rock64/releases/InRelease: The key(s) in the keyring /etc/apt/trusted.gpg.d/ayufan.key.chroot.gpg are ignored as the file has an unsupported filetype.
```

0.9.4: gitlab-ci-linux-build-146
disco-minimal-rock64-0.9.4-1146-arm64.img.xz

Just going to delete those chroot keys but just letting you know

**Comment by bvduijn** [comment]
Same issue in 0.9.14: gitlab-ci-linux-build-159

**Comment by jgonyea** [comment]
Still present in 0.9.16: gitlab-ci-linux-build-163

**Comment by ayufan** [comment]
This is fixed in 0.10.x.
